### ISSUE TITLE
DeulgeVPN does not run

### ISSUE THREAD

**Issue by KenMurphy** [Cloudbox/Community#106]
I installed DelugeVPN following the steps on the DelugeVPN page. The container is up and the subdomain has been created but I just get a 502 bad gateway when trying to visit it. A 'ps' on the server shows that OpenVPN is running as expected but there is no Deluge server running

**Comment by KenMurphy** [comment]
After some troubleshooting this appears to have been a multi-part issue. 

The first was my fault, I had the wrong VPN password configured so it prevented it from coming up and starting deluge.

The next issue is that after fixing #1 the script was failing while checking if the port was forwarded correctly. I changed STRICT_PORT_FORWARD to no in the container settings and this fixed that problem. Deluge was now launching!

Still couldn't reach it though. So #3 was that the container had joined the bridge network instead of the cloudbox network so nginx-proxy couldn't find it. I changed bridge to cloudbox in portainer and everything works correctly now.

Going to close this issue.
